changelog/unreleased/features/2284--parquet-store-plugin.md

```diff
@@ -0,0 +1,6 @@
+A new parquet store plugin allows VAST to store its data as parquet files,
+increasing storage efficiency at the expense of higher deserialization costs.
+Storage requirement for the archive (excluding indices) is reduced by 50%
+compared to the existing segment store with zstd compression enabled.
+CPU usage for suricata import is up ~ 10%,  mostly related to the more
+expensive serialization.
```

Comment:
Did you benchmark this again after switching to storing the list of table slices over the Arrow table?

Reply:
this measurement is only for the import, so the change shouldn't have an import. But thanks for the reminder; I intended to add some measurements for the read path as well, just didn't get to it yet.